### PR TITLE
Happy new year

### PR DESCRIPTION
**Pull request by bkchr**
Updates the copyright years and fixes wrong license headers.

### CODE REVIEW

**Review by kianenigma — APPROVED** [review]
🎉

**Review by shawntabrizi — APPROVED** [review]

File: HEADER [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // This file is part of Substrate.
```
Comment (andresilva) [inline HEADER:1]: It seems like this file isn't used anywhere. Can we rename it to `HEADER-APACHE` and also create a `HEADER-GPL`?

File: utils/prometheus/src/lib.rs [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
-// (at your option) any later version.
-
-// Substrate is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
-
-// You should have received a copy of the GNU General Public License
-// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+// Copyright (C) 2019-2021 Parity Technologies (UK) Ltd.
```
Comment (andresilva) [inline utils/prometheus/src/lib.rs:3]: Shouldn't this be GPL? It's not runtime code.

Comment (andresilva) [reply utils/prometheus/src/lib.rs:3]: Same for the rest of the files in this crate.

**Review by andresilva — CHANGES_REQUESTED** [review]
Happy new year 🤗

File: HEADER (hunk 1 above)
Comment (bkchr) [reply HEADER:1]: I don't understand this file either. We could also remove it 🤷

File: utils/prometheus/src/lib.rs (hunk 2 above)
Comment (bkchr) [reply utils/prometheus/src/lib.rs:3]: Idk 😂 I will change it

File: HEADER (hunk 1 above)
Comment (shawntabrizi) [reply HEADER:1]: It is for benchmarks template

Comment (bkchr) [reply HEADER:1]: @shawntabrizi isn't that `frame-weight-template.hbs`?

File: utils/prometheus/src/lib.rs (hunk 2 above)
Comment (bkchr) [reply utils/prometheus/src/lib.rs:3]: I checked it and it was APACHE before as well. Check `/prometheus/src/networking.rs`

File: HEADER (hunk 1 above)
Comment (shawntabrizi) [reply HEADER:1]: Ah yeah maybe I don't use it anymore :) None the less, isnt it good to have? Polkadot has one too

Comment (bkchr) [reply HEADER:1]: The point is that the header is different between the client/runtime.

I'm fine with keeping it for now.

File: utils/prometheus/src/lib.rs (hunk 2 above)
Comment (andresilva) [reply utils/prometheus/src/lib.rs:3]: Yeah, but it was wrong 🙈

Comment (bkchr) [reply utils/prometheus/src/lib.rs:3]: Primitives are also apache2

File: HEADER (hunk 1 above)
Comment (andresilva) [reply HEADER:1]: I assumed that this existed for people to copy/paste into files. That's why I suggested creating both files, like we have `LICENSE-GPL3` and `LICENSE-APACHE2`.

Comment (shawntabrizi) [reply HEADER:1]: That sounds super reasonable to me

Comment (bkchr) [reply HEADER:1]: Done.

**Comment by bkchr** [comment]
bot merge

**Pull request merged**